packages/valory/skills/decision_maker_abci/behaviours/reedem.py

```diff
@@ -966,6 +969,9 @@ def async_act(self) -> Generator:
             if not success:
                 return None
 
+            if self.synchronized_data.tx_submitter == BetPlacementRound.auto_round_id():
+                self.update_bet_transaction_information()
```

Comment:
Done.

Reply:
What I meant was, why are we checking `self.synchronized_data.tx_submitter == BetPlacementRound.auto_round_id()` here? It might not be intuitive for people who are not familiar with the tx settlement multiplexer and the trader's code.